app/scripts/controllers/swaps.ts

```diff
@@ -709,7 +709,7 @@ export default class SwapsController {
         : totalEthCost;
 
       const decimalAdjustedDestinationAmount = calcTokenAmount(
-        destinationAmount,
```

Comment:
It's expected that the destinationAmount of a Quote is either a string or null. If it was a null, null would be passed to calcTokenAmount. If it was undefined, it would have been a number type. This fixes the type and expected null handling